Library/Formula/git-octopus.rb

```diff
@@ -0,0 +1,11 @@
+class GitOctopus < Formula
+  homepage "https://github.com/lesfurets/git-octopus"
+  url "https://github.com/lesfurets/git-octopus/archive/v1.0.tar.gz"
+  def install
+    bin.install "bin/git-octopus"
+    man1.install "man/man1/git-octopus.1"
+  end
+  test do
```

Comment:
As much as Homebrew likes compact formulae, please don't squash the blocks together quite *this* much - Makes it incredibly difficult to read through, heh. An empty line between each major block is fine.